lib/jsdom/living/xmlhttprequest.js

```diff
@@ -186,25 +188,29 @@ module.exports = function createXMLHttpRequest(window) {
         return properties.responseCache;
       }
       let res = "";
+
+      const responseBuffer = (!properties.responseBuffer) ? null :
```

Comment:
No unnecessary parens. Here and below.

I am surprised ESLint did not catch this.